src/init.gradle

```diff
@@ -384,10 +426,37 @@ abstract class LicenseReportText extends DefaultTask {
         return licenseFiles
     }
 
+    boolean findUsableDepsFromTree(){
+        File depTreeFile = new File(depTreePath.get())
```

Comment:
:x: This is wrong. You must not have a `String` which represents a `File`. Therefore, `depTreePath` should be a `RegularFileProperty`, annotated with `@InputFile`

Reply:
I tried it initially, but got the error "An input file was expected to be present but it doesn't exist.", which is strange since I put @Optional, so it should not break anyway. My thinking was that because the dep tree file is made just before the task, and the file is being injected when configuring the task, maybe it is a problem there somewhere, but it just breaks for subprojects, the all.txt file is made just fine, so I am not sure what the problem is. My best guess is that the naming is off somehow

Reply:
Nvm, I tried it again, and it seems I just put an extra "/" before the filepath, so it broke. It works fine now